app/Console/Commands/GenerateAuthToken.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Laravel\Passport\ClientRepository;
+
+class GenerateAuthToken extends Command
+{
+    protected $signature = 'token:generate {user}';
+    protected $user;
+
+    public function __construct(User $user)
+    {
+        parent::__construct();
+        $this->user = $user;
+    }
+
+    public function handle()
+    {
+        try {
+            app()->make(ClientRepository::class)->personalAccessClient();
+        } catch (\RuntimeException $e) {
+            if ($e->getMessage() == 'Personal access client not found. Please create one.') {
```

Comment:
Is there not a more targeted exception for this? I'd like this defined as a constant rather than having raw strings in code if there isn't.

Reply:
Nope, passport just throws this runtime exception sadly, and there is no easier way for use to check if the personal access client has been enabled or not (that I could find)